### PR TITLE
Availability UI WIP

### PR DESCRIPTION
**Pull request by ajshort**
I started putting together a basic layout for the week availability UI. Opened this pull request to track progress on this. I've jammed it into the app and will try and tidy it up a bit over the next week.

### CODE REVIEW

**Comment by OSPFNeighbour** [comment]
Can you throw a screen shot up for me, save me building it. Pleeeeeease :p

**Comment by ajshort** [comment]
screenshot_1510488120

**Comment by sdunster** [comment]
looks awesome. let me know when it's ready to go on master

File: client/src/screens/week-availability.screen.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { Button, CheckBox, FlatList, StyleSheet, Text, View } from 'react-native';
+import { Divider, Icon, List } from 'react-native-elements';
```
Comment (sdunster) [inline client/src/screens/week-availability.screen.js]: `List` is unused.
You might also find that `react-native-elements` is unnecessary if you take a look at some of the other screens. I have imported an icons library already and I believe there's another way of doing dividers in some of the other views. I'm not opposed to adding more deps if they're useful though. :)

**Comment by sdunster** [comment]
Whats the plan here?

**Comment by ajshort** [comment]
I'll try and look at fleshing out the rest of this mockup by the end of the week.

**Comment by sdunster** [comment]
ping

**Pull request closed without merging**